minikube v1.16 with podman driver fails to start ( index out of range [1] with length 1)

<!--- Please include the "minikube start" command you used in your reproduction steps --->
**Steps to reproduce the issue:** 

Vagrantfile:


```bash
$script = <<SCRIPT
sudo apt install podman curl -y
curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
sudo dpkg -i minikube_latest_amd64.deb
minikube config set driver podman
sudo sysctl -w kernel.unprivileged_userns_clone=1
SCRIPT

Vagrant.configure("2") do |config|
  config.vm.box = "debian/testing64"
  config.vm.hostname = "bullseye-minikube"
  config.vm.provider :libvirt do |v|
    v.memory = 4096 
    v.cpus   = 4
    # v.cpu_feature :name => 'vmx', :policy => 'require'
  end
  config.vm.provision "shell", inline: $script
end
```





<!--- TIP: Add the "--alsologtostderr" flag to the command-line for more logs --->
**Full output of failed command:** 



**Full output of `minikube start` command used, if not already included:**

```bash
vagrant ssh
vagrant@bullseye-minikube:~$ minikube start --driver podman
😄  minikube v1.16.0 on Debian bullseye/sid
✨  Using the podman (experimental) driver based on user configuration
👍  Starting control plane node minikube in cluster minikube
💾  Downloading Kubernetes v1.20.0 preload ...
    > preloaded-images-k8s-v8-v1....: 491.00 MiB / 491.00 MiB  100.00% 6.34 MiB
🔥  Creating podman container (CPUs=2, Memory=2200MB) ...
panic: runtime error: index out of range [1] with length 1

goroutine 187 [running]:
k8s.io/minikube/pkg/drivers/kic/oci.podmanNetworkInspect(0x1cefa53, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x2046220, ...)
	/usr/local/google/home/tstromberg/src/minikube/pkg/drivers/kic/oci/network_create.go:222 +0x6c1
k8s.io/minikube/pkg/drivers/kic/oci.containerNetworkInspect(0xc0004fae90, 0x6, 0x1cefa53, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/usr/local/google/home/tstromberg/src/minikube/pkg/drivers/kic/oci/network_create.go:155 +0x187
k8s.io/minikube/pkg/drivers/kic/oci.CreateNetwork(0xc0004fae90, 0x6, 0xc0004fae20, 0x8, 0x2, 0xc000d43190, 0x6, 0x0, 0x47b758)
	/usr/local/google/home/tstromberg/src/minikube/pkg/drivers/kic/oci/network_create.go:65 +0x1d0
k8s.io/minikube/pkg/drivers/kic.(*Driver).Create(0xc00022bd40, 0xc0009beb20, 0x1d)
	/usr/local/google/home/tstromberg/src/minikube/pkg/drivers/kic/kic.go:89 +0x4c5
k8s.io/minikube/pkg/minikube/machine.(*LocalClient).Create(0xc0009101c0, 0xc000554720, 0x0, 0x0)
	/usr/local/google/home/tstromberg/src/minikube/pkg/minikube/machine/client.go:224 +0x48f
k8s.io/minikube/pkg/minikube/machine.timedCreateHost.func2(0x20b8fe0, 0xc0009101c0, 0xc000554720, 0xc000438940, 0xc0001dcaf0)
	/usr/local/google/home/tstromberg/src/minikube/pkg/minikube/machine/start.go:194 +0x3b
created by k8s.io/minikube/pkg/minikube/machine.timedCreateHost
	/usr/local/google/home/tstromberg/src/minikube/pkg/minikube/machine/start.go:193 +0x107
```

Podman version:
```bash
sudo apt show podman
Package: podman
Version: 2.1.1+dfsg1-3
```

In general podman works fine:
```bash
vagrant@bullseye-minikube:~$ podman pull docker.io/library/alpine:latest
vagrant@bullseye-minikube:~$ podman run alpine uname -a
Linux b15c5a529f04 5.6.0-2-amd64 #1 SMP Debian 5.6.14-1 (2020-05-23) x86_64 Linux
```



**Optional: Full output of `minikube logs` command:**

```bash
vagrant@bullseye-minikube:~$ minikube logs

❌  Exiting due to GUEST_STATUS: state: unknown state "minikube": sudo -n podman container inspect minikube --format=: exit status 125
stdout:

stderr:
Error: error inspecting object: no such container minikube


😿  If the above advice does not help, please let us know: 
👉  https://github.com/kubernetes/minikube/issues/new/choose
```

It looks for an existing previous network, doesn't find anyone - and then goes right on with crashing...

```
I0104 18:09:26.542911    9053 cli_runner.go:111] Run: sudo -n podman network inspect minikube --format "{{range .plugins}}{{if eq .type "bridge"}}{{(index (index .ipam.ranges 0) 0).subnet}},{{(index (index .ipam.ranges 0) 0).gateway}}{{end}}{{end}}"
W0104 18:09:26.603257    9053 cli_runner.go:149] sudo -n podman network inspect minikube --format "{{range .plugins}}{{if eq .type "bridge"}}{{(index (index .ipam.ranges 0) 0).subnet}},{{(index (index .ipam.ranges 0) 0).gateway}}{{end}}{{end}}" returned with exit code 125
I0104 18:09:26.603483    9053 network_create.go:235] running [podman network inspect minikube] to gather additional debugging logs...
I0104 18:09:26.603520    9053 cli_runner.go:111] Run: sudo -n podman network inspect minikube
W0104 18:09:26.666232    9053 cli_runner.go:149] sudo -n podman network inspect minikube returned with exit code 125
I0104 18:09:26.666275    9053 network_create.go:238] error running [sudo -n podman network inspect minikube]: sudo -n podman network inspect minikube: exit status 125
stdout:

stderr:
Error: unable to find network configuration for minikube: network not found
I0104 18:09:26.666306    9053 network_create.go:240] output of [sudo -n podman network inspect minikube]: 
** stderr ** 
Error: unable to find network configuration for minikube: network not found

** /stderr **
I0104 18:09:26.666376    9053 cli_runner.go:111] Run: sudo -n podman network inspect podman --format "{{range .plugins}}{{if eq .type "bridge"}}{{(index (index .ipam.ranges 0) 0).subnet}},{{(index (index .ipam.ranges 0) 0).gateway}}{{end}}{{end}}"
I0104 18:09:26.723395    9053 client.go:168] LocalClient.Create took 181.117027ms
panic: runtime error: index out of range [1] with length 1
```

--- a/pkg/drivers/kic/oci/network_create.go
+++ b/pkg/drivers/kic/oci/network_create.go
@@ -221,8 +221,13 @@ func podmanNetworkInspect(name string) (netInfo, error) {
 		return info, err
 	}
 
+	output := rr.Stdout.String()
+	if output == "" {
+		return info, fmt.Errorf("no bridge network found for %s", name)
+	}
+
 	// results looks like 172.17.0.0/16,172.17.0.1,1500
-	vals := strings.Split(strings.TrimSpace(rr.Stdout.String()), ",")
+	vals := strings.Split(strings.TrimSpace(output), ",")
 	if len(vals) == 0 {
 		return info, fmt.Errorf("empty list network inspect: %q", rr.Output())
 	}

